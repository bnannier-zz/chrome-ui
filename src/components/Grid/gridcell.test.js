import React from "react";
import { configure, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import { Grid, GridCell } from "../";

describe("GridCell Tests", () => {
  it("Default", function() {
    const wrapper = mount(
      <Grid>
        <GridCell>test</GridCell>
      </Grid>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
