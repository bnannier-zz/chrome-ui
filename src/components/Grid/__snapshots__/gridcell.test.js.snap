// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridCell Tests Default 1`] = `
<Grid>
  <Base>
    <div
      className="mdc-layout-grid"
    >
      <GridInner>
        <Base>
          <div
            className="mdc-layout-grid__inner"
          >
            <GridCell>
              <Base>
                <div
                  className="mdc-layout-grid__cell"
                >
                  test
                </div>
              </Base>
            </GridCell>
          </div>
        </Base>
      </GridInner>
    </div>
  </Base>
</Grid>
`;
